content/project-organization/quality-council/processes/pr-management.md

```diff
@@ -0,0 +1,159 @@
+# Testing Pull Request
+
+
+## How to test a PR manually
+
+You want to take care of a PR and you don't know how to handle it, or you've simply forgotten how. No problem, this github page is for you. 
+
+
+### 1. Taking information from a PR
+
+
+One of the first things to do when starting a PR is to find out as much as possible about it. To do this, you'll find this table at the start of each PR
+
+![img.png](images/readme7.png)
+
+To begin with, we have the **Branch**, which will be useful for preparing our shop and automated tests. So we can already set up a shop accordingly if we don't already have a shop in this branch. 
+
+To do this, go to your terminal and execute the commands below: 
+
+
+ℹ️ For your information, I recommend that you put it in the *Computer/var/www/html* folder.
+
+![img.png](images/readme8.png)
+
+
+We can see that in **violet** we've put the "**Branch "** we're interested in and in **brown** the name of our file, which can be modified as we wish. 
+
+Still with this **Branch**, I advise you to launch the automated Tests. To do this, I suggest you look at the end of this chapter dedicated to it.
+
+
+### 2. Replicating the issue
+
+Now that we have an installed shop in which the issue is still present, you can check the three elements of the table: 
+
+- Description: Sometimes, the description will tell you where the PR is heading, but it can also tell you where it may have affected the code and which axes we'll be able to check next.
+- How to Test: When this field is filled in, it enables you to easily identify a way of reproducing the issue. Often, it will simply refer to the issue linked to the PR.
+- Fixed issue or discussion: When you click on the indicated issue, you'll have all the information you need to reproduce it.
+
+Once on the issue, all you have to do is scroll down to find the "Steps to reproduce" section and follow the steps one by one, taking the example of our same PR:
+
+![img.png](images/readme9.png)
+
+If the issue has been validated by our testers, ( ⚠️ any PR that mentions an issue must have this issue validated by the QA, otherwise the PR will not be taken into account ) the steps to reproduce must have been reviewed by them and is therefore easily reproducible. 
```

Comment:
```suggestion
If the issue has been validated by our testers, (⚠️ any PR that mentions an issue must have this issue validated by the QA, otherwise the PR will not be taken into account) the steps to reproduce must have been reviewed by them and are therefore easily reproducible. 
```